CMakeLists.txt

```diff
@@ -152,13 +151,22 @@ string(TOLOWER ${CMAKE_SYSTEM_NAME} TJS_PLATFORM)
 target_compile_options(tjs PRIVATE ${tjs_cflags})
 target_compile_definitions(tjs PRIVATE TJS__PLATFORM="${TJS_PLATFORM}")
 target_include_directories(tjs PRIVATE ${CURL_INCLUDE_DIRS})
+target_include_directories(tjs PUBLIC src)
 target_link_libraries(tjs qjs uv_a m3 sqlite3 m pthread ${CURL_LIBRARIES})
 
 if (BUILD_WITH_MIMALLOC)
     target_compile_definitions(tjs PRIVATE TJS__HAS_MIMALLOC)
     target_link_libraries(tjs mimalloc-static)
 endif()
 
+add_executable(tjs-cli EXCLUDE_FROM_ALL
+    src/cli.c
+)
+target_link_libraries(tjs-cli tjs)
```

Comment:
Don't we need all the linked libraries here instead?

Reply:
For me, it works without specifying all the libraries.
I think we can either:
- leave it as it is now
- add the `PUBLIC` scope for the `target_link_libraries` of the tjs library
- add all the libraries here again

What do you prefer?

Reply:
Let's leave it for now then.